src/uint256.h

```diff
@@ -144,4 +144,6 @@ inline uint256 uint256S(const std::string& str)
     return rv;
 }
 
+uint256& UINT256_ONE();
```

Comment:
Why not simply make this `extern uint256 UINT256_ONE;` instead of a singleton?

Reply:
Clang didn't like it. Something about static initialization order resulting in a potential uninitialized object access.

Reply:
Weird. :/ I've seen that once before too. Sounds like a compiler bug tbh.